RUMM-621 Update Tracing docs with auto instrumentation info (#200)
DataDog/dd-sdk-ios: docs/trace_collection.md

        request.addValue(value, forHTTPHeaderField: headerField)
    }
    ```
    This will set additional tracing headers on your request, so that your backend can extract it and continue distributed tracing. Once the request is done, within a completion handler, call `span.finish()`. If your backend is also instrumented with [Datadog APM & Distributed Tracing][10] you will see the entire front-to-back trace in Datadog dashboard.

>     * The SDK will automatically trace all network requests made to given hosts if `tracedHosts` array is specified during Datadog initialization:

Reviewer: ```suggestion
    * To have the SDK automatically trace all network requests made to given hosts, specify `tracedHosts` array during Datadog initialization:
```

    ```swift
    Datadog.initialize(
        appContext: .init(),
        configuration: Datadog.Configuration
            .builderUsing(clientToken: "<client_token>", environment: "<environment_name>")
            .set(tracedHosts: ["example.com", "api.yourdomain.com"])
            .build()
    )
    ```

>     This will trace all requests made to `example.com` and `api.yourdomain.com` (e.g. `https://api.yourdomain.com/v2/users` or `https://subdomain.example.com/image.png`).

Reviewer: ```suggestion
    This will trace all requests made to `example.com` and `api.yourdomain.com` (for example, `https://api.yourdomain.com/v2/users` or `https://subdomain.example.com/image.png`).
```
